lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -26,6 +26,7 @@
 /* globals getFilenamePrefix Util */
 
 /** @typedef {import('./dom.js')} DOM */
+/** @typedef {import('./util.js')} Util */
```

Comment:
doesn't the `html-renderer.d.ts` take care of this? (I don't think either typedef is needed)

Reply:
ya, removed